ui/components/src/utils/theme-gen.ts

```diff
@@ -172,6 +172,22 @@ export function generateChartsTheme(
         fontSize: 11,
       },
     },
+    axisPointer: {
+      lineStyle: {
+        color: muiTheme.palette.grey[500],
+      },
+    },
+    markLine: {
+      symbol: 'none',
+      symbolSize: 0,
+      itemStyle: {
+        color: muiTheme.palette.grey[700],
```

Comment:
i think we should use a different color, something more transparent?

right now it seems too bold in light mode

![image](https://github.com/perses/perses/assets/9817826/142b5b55-8f92-4c68-8954-7dbb9faebfe0)